telegram/ext/messagehandler.py

```diff
@@ -66,9 +68,13 @@ def __init__(self,
                  pass_user_data=False,
                  pass_chat_data=False,
                  message_updates=True,
-                 channel_post_updates=True):
-        if not message_updates and not channel_post_updates:
-            raise ValueError('Both message_updates & channel_post_updates are False')
+                 channel_post_updates=True,
+                 edited_updates=False):
+        if not message_updates and not channel_post_updates and not edited_updates:
+            raise ValueError(
+                'message_updates, channel_post_updates and edited_updates are all False')
+        if allow_edited:
+            warnings.warn('allow_edited is getting deprecated, please use edited_updates instead')
```

Comment:
Please don't... if you rename an argument, then it is sooo much nicer to also rename the attribute in the class while you are at it. Then in the future when we wanna remove the deprecated stuff, we don't have to change a ton of stuff.